src/compiler/fbc.bas

```diff
@@ -3097,12 +3113,18 @@ private sub hAddDefaultLibs( )
 			    defined(__FB_NETBSD__)
 				fbcAddDefLibPath( "/usr/X11R6/lib" )
 			#endif
+			
+			#if defined(__FB_DARWIN__) and defined(ENABLE_XQUARTZ)
+				fbcAddDefLibPAth( "/opt/X11/lib" )
+			#endif
 
-			fbcAddDefLib( "X11" )
-			fbcAddDefLib( "Xext" )
-			fbcAddDefLib( "Xpm" )
-			fbcAddDefLib( "Xrandr" )
-			fbcAddDefLib( "Xrender" )
+			#if not(defined(__FB_DARWIN__)) or defined(ENABLE_XQUARTZ)
```

Comment:
Here the `not` operator should probably be put inside the parentheses:
`#if (not defined(__FB_DARWIN__)) or defined(ENABLE_XQUARTZ)`
so it's clear that it should affect only the one `defined()`, not the entire `or` operation. I'm never sure about `not` operator precedence...